ngraph/python/src/ngraph/opset3/__init__.py

```diff
@@ -114,7 +114,6 @@
 from ngraph.opset1.ops import relu
 from ngraph.opset1.ops import reshape
 from ngraph.opset1.ops import result
-from ngraph.opset3.ops import reverse
```

Comment:
Does this lists with operations in inopsetX/__init__.py files match C++ opsetX_tbl.hpp ?

Reply:
Opsets in pyAPI represent lists of operations from the specification in `docs/ops/opsetX.md`.

Reply:
@lazarevevgeny I remember you said that list of ops in ngraph opset is a reference list. So what should pyAPI follow? spec or list of ops in _tbl.hpp  files?

Reply:
The pyAPI should follow the opsetX_tbl.hpp file, not the documentation